src/main/java/com/spikes2212/command/genericsubsystem/commands/AccelerateGenericSubsystem.java

```diff
@@ -26,7 +26,6 @@
      * time.
```

Comment:
document all the fields above this line:

- time
- acceleration
- currentSpeed
- startTime